### PR TITLE
chore: cleanup `.prettierignore` file

### PR DESCRIPTION
**Pull request by Gudahtt**
## **Description**

A few invalid entries have been removed from the `.prettierignore` file, and the remaining entries have been alphabetized so that they are easier to quickly scan.

I did this because I was investigating a strange ESLint error, and these entries made the investigation a little more complicated than it needed to be.

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/24828?quickstart=1)

## **Related issues**

N/A

## **Manual testing steps**

N/A

## **Screenshots/Recordings**

N/A

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -1,18 +1,16 @@
-node_modules/**/*
-lavamoat/**/policy.json
-dist/**/*
-builds/**/*
-test-*/**/*
```
Comment (Gudahtt) [inline .prettierignore:5]: This was split into these two lines:

```
/test-artifacts/**/*
/test/test-results/**/*
```

File: .prettierignore [hunk 2]
```diff
@@ -1,18 +1,16 @@
-node_modules/**/*
```
Comment (Gudahtt) [inline .prettierignore:1]: This was listed twice, you can see that the second entry is preserved in this diff

**Comment by Gudahtt** [comment]
I wasn't sure if ESLint would treat the leading slash in the same way (e.g. leading slash means it's specifically in the project root, no leading slash matches the path in any subdirectory or in the root), the docs weren't clear on the subject.

But I tested it out by adding test files in the root and in a subdirectory, each with a lint error. And I validated that it treats them the same way.

My test was to create `test1.js`, `test2.js`, and `test3.js`, each with a lint error. And then duplicate them in a directory as well. Then ignore `test1.js` and `/test2.js`. I expected to see lint errors for both `test3.js` files (the control), no lint errors for `test1.js`, and lint errors just for the `test2.js` in a directory. And that's exactly what I saw.

**Review by vthomas13 — APPROVED** [review]

File: .prettierignore [hunk 3]
```diff
@@ -1,18 +1,16 @@
...
 *.scss
-development/chromereload.js
-development/ts-migration-dashboard/filesToConvert.json
-development/ts-migration-dashboard/build/**
-public/**/*
-development/charts/**
+.nyc_output/**/*
 node_modules/**/*
-storybook-build/**/*
-jest-coverage/**/*
```
[the remark this answers is not in the document]
Comment (HowardBraham) [reply .prettierignore:17]: The `storybook-build` folder exists after you run `yarn storybook:build` 😄

[the remark this answers is not in the document]
Comment (Gudahtt) [reply .prettierignore:17]: This comment was in reference to `jest-coverage`. The `storybook-build` line was preserved, it is below

[the remark this answers is not in the document]
Comment (HowardBraham) [reply .prettierignore:17]: Ohhhhh!  Nevermind 😄

**Review by HowardBraham — APPROVED** [review]

**Comment by Gudahtt** [comment]
Hmm, I am having trouble reproducing that. It seems unlikely that it would be caused by this change as well.

**Pull request merged**